package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@ironcorelabs/ironnode",
-    "version": "0.8.3",
+    "version": "0.8.4-pre",
```

Comment:
I think this is the correct format so that `bump-version` is happy?

Reply:
Yes - that should satisfy it.